tools/profile.py

```diff
@@ -81,9 +88,9 @@ def stack_id_err(stack_id):
     formatter_class=argparse.RawDescriptionHelpFormatter,
     epilog=examples)
 thread_group = parser.add_mutually_exclusive_group()
-thread_group.add_argument("-p", "--pid", type=positive_int,
+thread_group.add_argument("-p", "--pid", type=positive_int_list,
     help="profile process with this PID only")
```

Comment:
could you change this help to 'profile process with one or more comma separated PIDs only'